radixdlt/src/main/java/com/radixdlt/consensus/liveness/Pacemaker.java

```diff
@@ -17,37 +17,53 @@
 
 package com.radixdlt.consensus.liveness;
 
-import com.radixdlt.consensus.NewView;
+import java.util.Optional;
+
+import com.radixdlt.consensus.Proposal;
+import com.radixdlt.consensus.QuorumCertificate;
+import com.radixdlt.consensus.ViewTimeout;
+import com.radixdlt.consensus.Vote;
 import com.radixdlt.consensus.HighQC;
 import com.radixdlt.consensus.bft.View;
 
-import com.radixdlt.consensus.bft.BFTValidatorSet;
-import java.util.Optional;
-
 /**
  * Manages the pacemaker state machine.
  */
 public interface Pacemaker extends PacemakerState {
 	/**
-	 * Signifies to the pacemaker that a timeout for a given view is processed
-	 * @param view the view to timeout
+	 * Signifies to the pacemaker that a vote has been received.
+	 *
+	 * @param vote the vote received
+	 * @return an {@link Optional} {@link QuorumCertificate} if a
+	 * 		quorum was formed with this vote
 	 */
-	void processLocalTimeout(View view);
+	Optional<QuorumCertificate> processVote(Vote vote);
 
 	/**
-	 * Signifies to the pacemaker that a quorum has agreed that a view has been completed
-	 * @param syncInfo the sync info
+	 * Signifies to the pacemaker that a proposal has been received.
+	 *
+	 * @param proposal the proposal received
 	 */
-	void processQC(HighQC syncInfo);
+	void processProposal(Proposal proposal);
 
-	// FIXME: Temporary
-	void processNextView(View view);
+	// FIXME: To be removed when TCs implemented
+	void processViewTimeout(ViewTimeout viewTimeout);
+
+	/**
+	 * Processes a local timeout, causing the pacemaker to move to the next
+	 * view if the timeout is for the current view.
```

Comment:
Would re-word this in that pacemaker only moves to next view on quorum. Otherwise, pacemaker stays in same view.

Reply:
Yep, oops.  Will fix.